opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -211,6 +220,20 @@ class BlackOil : public BaseFluidSystem<Scalar, BlackOil<Scalar> >
         assert(numActivePhases_ >= 2 && numActivePhases_ <= 3);
 
         setEnableDissolvedGas(deck.hasKeyword("DISGAS"));
+        if(deck.hasKeyword("DRSDT")){
```

Comment:
since this is a time related quantity, this does not belong into the fluid system. (fluid systems should only be concerned with thermodynamics.)